test/util/JoinHelpers.h

```diff
@@ -47,3 +49,27 @@ IdTable useJoinFunctionOnIdTables(const IdTableAndJoinColumn& tableA,
 
   return result;
 }
+
+/*
+ * @brief Returns a lambda for calling the Join::hashJoin.
```

Comment:
```suggestion
 * @brief Returns a lambda for calling `Join::hashJoin` via `ad_utility::callFixedSize`.
```

Reply:
Of course adapt the comment blow similarly.

Reply:
Done, see [f38f86f](https://github.com/ad-freiburg/qlever/pull/860/commits/f38f86f9df80c08d6f7176228e322cc6e9fce1fd).